actionpack/lib/action_dispatch/http/url.rb

```diff
@@ -1,26 +1,36 @@
 # frozen_string_literal: true
 
 require "active_support/core_ext/module/attribute_accessors"
+require "action_dispatch/http/uri"
```

Comment:
it is, without it we get an 

```
NameError: uninitialized constant ActionDispatch::Http::URI
    /workspaces/rails/actionpack/lib/action_dispatch/http/url.rb:65:
```

Reply:
Is that because `bootstrap.rb` is directly requiring `url.rb` instead of requiring `action_dispatch` and then relying upon autoloads? Does the rest of the backtrace carry any clues?

Not hugely important, just moderately odd that we'd need to explicitly require something we're defining an autoload for.